Library/Formula/giflossy.rb

```diff
@@ -0,0 +1,33 @@
+class Giflossy < Formula
+  desc "Lossy LZW compression animated GIF reduce file sizes by 30%—50%"
```

Comment:
Can we tweak this a bit? It doesn't read entirely coherently at the moment. Maybe something like:

```ruby
desc "Lossy LZW compression, reduces GIF file sizes by 30-50%"
```

Or something?

Not sure what the policy is on commas in descriptions.